app/views/posts/show.html.erb

```diff
@@ -1 +1,13 @@
-<h2>Here is a list of posts for a given user</h2>
\ No newline at end of file
+<h2>Specific post and its comment</h2>
+<div class="posts">
+  <div class="card-info">
+    <h3>Title : <%=@posts.title%></h3>
+    <p>Post : <%= @posts.text%> </p>
+    <p class="stats-comment"> Comments: <%= @posts.comments_counter or 0 %> | Likes: <%= @posts.likes_counter or 0 %> </p>
+  </div>
+  <div class="comment-container">
+    <% @posts.comments.each do |comment| %>
+      <p> <%= comment.user.name%>: <%= comment.text%><p>
+    <% end %>  
+  </div>
+</div>
```

Comment:
- [ ] Kindly improve the layout of your post show page to match the wireframe. As you can the placements of some items are quite different from the one on the wireframe.

- [ ] You also seem to be missing the post author details.

  Your Implementation | Wireframe design
  :---:|:---:
  ![bcg](https://user-images.githubusercontent.com/54848363/198068391-a2414010-2d24-492a-85cd-11851ac45968.jpg) | ![blog_single_post](https://user-images.githubusercontent.com/54848363/198068714-ea99fee2-3f1b-445f-8330-027d3eaca328.jpg)

Reply:
Done as requested, thank you 👍🏾